cmd/data-crawler.go

```diff
@@ -626,3 +626,11 @@ func sleepDuration(d time.Duration, x float64) {
 		time.Sleep(d)
 	}
 }
+
+// healReplication will heal a scanned item that has failed replication.
+func (i *crawlItem) healReplication(ctx context.Context, o ObjectLayer, meta actionMeta) {
+	if meta.oi.ReplicationStatus != replication.Failed {
```

Comment:
would a switch case be better? for future states?

Reply:
unless we plan to extend the API, this is the only status that needs to be healed.